v22.1/ui-queues-dashboard.md

```diff
@@ -0,0 +1,189 @@
+
+The **Split Queue** graph shows you various details about the health and performance of the split queue.
+
+On hovering over the graph, the values for the following metrics are displayed:
+
+Metric | Description
+--------|----
+Successful Actions | The number of replicas successfully processed by the split queue, as tracked by the `queue.split.process.success` metric.
+Pending Actions | The number of pending replicas in the split queue, as tracked by the `queue.split.pending` metric.
+
+## Merge Queue
+
+<img src="{{ 'images/v22.1/ui_merge_queue.png' | relative_url }}" alt="DB Console merge queue graph" style="border:1px solid #eee;max-width:100%" />
+
+The **Merge Queue** graph shows you various details about the health and performance of the merge queue.
+
+On hovering over the graph, the values for the following metrics are displayed:
+
+Metric | Description
+--------|----
+Successful Actions | The number of replicas successfully processed by the merge queue, as tracked by the `queue.merge.process.success` metric.
+Pending Actions | The number of pending replicas in the merge queue, as tracked by the `queue.merge.pending` metric.
+
+## GC Queue
+
+<img src="{{ 'images/v22.1/ui_gc_queue.png' | relative_url }}" alt="DB Console GC queue graph" style="border:1px solid #eee;max-width:100%" />
+
+The **GC Queue** graph shows you various details about the health and performance of the garbage collection queue.
+
+On hovering over the graph, the values for the following metrics are displayed:
+
+Metric | Description
+--------|----
+Successful Actions | The number of replicas successfully processed by the garbage collection queue, as tracked by the `queue.gc.process.success` metric.
+Pending Actions | The number of pending replicas in the garbage collection queue, as tracked by the `queue.gc.pending` metric.
+
+## Raft Log Queue
+
+<img src="{{ 'images/v22.1/ui_raftlog_queue.png' | relative_url }}" alt="DB Console Raft log queue graph" style="border:1px solid #eee;max-width:100%" />
+
+The **Raft Log Queue** graph shows you various details about the health and performance of the Raft log queue.
+
+On hovering over the graph, the values for the following metrics are displayed:
+
+Metric | Description
+--------|----
+Successful Actions | The number of replicas successfully processed by the Raft log queue, as tracked by the `queue.raftlog.process.success` metric.
+Pending Actions | The number of pending replicas in the Raft log queue, as tracked by the `queue.raftlog.pending` metric.
+
+## Raft Snapshot Queue
+
+<img src="{{ 'images/v22.1/ui_raftsnapshot_queue.png' | relative_url }}" alt="DB Console Raft snapshot queue graph" style="border:1px solid #eee;max-width:100%" />
+
+The **Raft Snapshot Queue** graph shows you various details about the health and performance of the Raft repair queue.
+
+On hovering over the graph, the values for the following metrics are displayed:
+
```

Comment:
Same suggestion as the previous table, might be more readable with the full metric name split into its own column so they are aligned vertically.